presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculator.java

```diff
@@ -50,6 +53,8 @@
 @Test(singleThreaded = true)
 public class TestFilterStatsCalculator
 {
+    private static final VarcharType MEDIUM_VARCHAR_TYPE = VarcharType.createVarcharType(100);
```

Comment:
what's medium about this varchar? is it the size?

Reply:
Yes, this is varchar of a medium size.